Job/DoctrineJobRepository.php

```diff
@@ -65,9 +71,20 @@ public function getJobManager()
     /**
      * {@inheritdoc}
      */
+    public function getJobExecution(JobInstance $jobInstance)
```

Comment:
Okay ! So in this case you can remove the setJobInstance at line 88 (in createJobExecution)